src/Connection.js

```diff
@@ -18,7 +18,9 @@ export default class Connection {
 
         if (handle[key].is_pool) {
             // handle._logger.log('Added a pool connection for', key);
-            connection = mysql.createPool(handle[key].config);
+            connection = !handle[key].connection
+                       ? mysql.createPool(handle[key].config)
+                       : handle[key].connection;
```

Comment:
that is mostly because they both have the `.query` function.

Created an issue here https://github.com/anyTV/anytv-node-mysql/issues/12